MAUI/ImageEditor/getting-started.md

```diff
@@ -253,4 +253,58 @@ The [`GetImageStream`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Image
     }
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Get the Image Original Size
+The [`OriginalImageSize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_OriginalImageSize) property is used to get the image's original size.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Text="Get Image Original Size"
+                Clicked="OnGetImageOriginalSize" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnGetImageOriginalSize(object sender, EventArgs e)
+    {
+        Size originalSize = this.imageEditor.OriginalImageSize;
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+N> The size value will be available only after image is loaded into view.
+
+## Get the Image Rendered Size
+The image editor utilizes the `AspectFit` image scaling of Image control, so that the entire image fits into the display area, with blank space added to the top or bottom sides depending on whether the image is wide or tall. The [`ImageRenderedSize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ImageRenderedSize) property is used to get the current rendered size of the image inside the display area.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Text="Get Image Rendered Size"
+                Clicked="OnGetImageRenderedSize" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnGetImageRenderedSize(object sender, EventArgs e)
+    {
+        Size imageSize = this.imageEditor.ImageRenderedSize;
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+N> The size value will be available only after image is loaded into view.
```

Comment:
N> The size value will only be available after the image has been loaded into view.